### ISSUE TITLE
Migrating RDS MySQL to Aurora Limitations

### ISSUE THREAD

**Issue by cooper667** [awsdocs/amazon-rds-user-guide#8]
If you try to migrate a snapshot or exiting RDS MySQL database to Aurora that has options set that Aurora doesn't support (my example is Memcached, there may be others), it will fail and result in a broken cluster. Support didn't point me in the right direction, they assumed the snapshot was corrupted, but removing the offending options solves it.

Is this worth noting somewhere as it would have saved me a few hours of pain and back and forths with support? 

(OR did it just miss it somewhere?)

**Comment by randyurbano** [comment]
Thanks for your feedback. I am investigating limitations for migrating RDS MySQL snapshots to Aurora MySQL.

**Comment by randyurbano** [comment]
You are correct that the memcached option is not supported when migrating from MySQL to Aurora MySQL. That was the only unsupported option that development could identify. I will update the AWS RDS documentation to include this limitation. Thanks again for your feedback.
